.circleci/config.yml

```diff
@@ -270,6 +334,18 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - make-rpm:
```

Comment:
What's the intended logic with the RPM/Deb building/publishing?

Build packages in all branches, but only push for tags starting with `v` and `master`?

Reply:
@j-white @RangerRick We do releases right now manually we haven't really talked about how we want to have the release logic. I would suggest following the approach from Alec where we publish release artifacts on git version tags prefixed with `v`, develop and release branches:

```yaml
filters:
  tags:
    only: /^v.*/
      branches:
        only:
          - develop
          - /release-.*/
```
It gives us a little bit more flexibility. We could do also something more restrictive and just publish release artifacts on git version tags:

```yaml
filters:
  tags:
    only: /^v.*/
      branches:
        ignore: /.*/
```
I'm open for one way or another. We can decide this here in the comments and fix the PR. I'll extract the release workflow from this conversation and we can it to the docs.